### PR TITLE
mention the Rocq rename

### PR DESCRIPTION
**Pull request by gasche**
The Coq team announced a rename months ago, but there is no sign of it on the webpage. This is silly!

The present PR adds just a couple mentions of the new Rocq name, enough for people to know that this is happening for real -- but not enough that it is a lot of work or that tricky questions have to be answered for consistency.

The proposed explanation includes an invitation to use the combined name "Coq/Rocq" during a transition period, so that people that only know of the old name (that is, almost everyone) does not get lost. This is used on the webpage itself, whose title is now "The Coq/Rocq proof assistant."

### CODE REVIEW

File: incl/header.html
Comment (silene) [inline incl/header.html:6]: Keep in mind that Inria has an official trademark on the name Coq (which is recognized in both USA and China, for what that is worth). I have only rudimentary notions on what that implies and it would have to be double-checked with Inria lawyers, but I am pretty sure that such a prominent modification is the fastest way to lose the trademark on Coq.

Comment (gasche) [reply incl/header.html:6]: My understanding (not a lawyer, etc.):
- Existing versions of Coq released under that name are covered by a Coq trademark and would remain so after a rename.
- Trademark rights may be lost if the trademark is not used anymore for a period of time (Wikipedia says [five years](https://en.wikipedia.org/wiki/Trademark#Maintaining_rights)), so (1) any change would only take effect after a few years, when presumably a rename would be deeply in progress and (2) if the website uses a dual naming scheme, Coq *and* Rocq, then we are still using the trademark and the rights may well be maintained.

Who has asked INRIA lawyers about the rename, and what did they say?

Comment (silene) [reply incl/header.html:6]: Writing somewhere on the website that "the Coq proof assistant is also known as the Rocq proof assistant" is presumably fine. Writing "the Coq/Rocq proof assistant" in the title of every single page of the website is no longer a case of dual naming. It is pure single naming.

That said, I have always been a proponent of keeping the Coq prefix in the new name (e.g., Coqatrix) to ease the transition process, but the idea never got enough traction. So, having the tool be renamed Coq/Rocq (Coqroach?) would certainly have my favor. But this ship has long sailed and the plan agreed upon is to get Coq to be renamed into Rocq, not into Coq/Rocq.

As for disclosing on a public forum what was discussed with the lawyers, I would not dare. I suggest you directly phone our [coordinator](https://coq.inria.fr/coq-team.html) to get a feel of how far the lawyers have progressed on the topic.

Comment (gasche) [reply incl/header.html:6]: I don't think that people would confuse Coq/Rocq for a single name, but I am happy to try to find a wording that would please you, in that it is clear that Coq and Rocq are both acceptable names for the system.

I don't particularly care about trademarks for open source project myself -- I think that it's probably a complete waste of time and effort -- and I am happy to ignore anything related to them, instead of having to phone people to learn secret information about the project. It is perfectly fine if the Coq stakeholders decide that trademark protection is important to them and they do stuff along those lines, as long as it remains orthogonal to the healthy development of the project. Deciding a rename, involving the community in a major way to decide a better name, officially announcing that a choice has been made, *and then doing nothing externally visible at all for months* is not healthy behavior for an open-source project.

Comment (gasche) [reply incl/header.html]: I just pushed a new version. Instead of using "The Coq/Rocq Proof Assistant", the new version says "The Coq (Rocq) Proof Assistant". Initially I wrote "The Coq Proof Assistant, the Rocq Prover", but this was too large and the banner would overflow on larger zoom level.

The wording in the right menu also does not use Coq/Rocq anymore:

> During this transition period, please feel free to use both names, for example by writing "formalized with Rocq (formerly Coq)" or "formalized with Coq (also named Rocq)".

Comment (herbelin) [reply incl/header.html]: @tabareau evaluates the estimal arrival time to December. Maybe better to stay on the safe side and be explicit on that, in the style: "planned to be renamed the Rocq prover by the end of 2024".

Comment (gasche) [reply incl/header.html]: Here is what I had before (in full):

> October 2023: we have decided to rename 'Coq' into 'The Rocq Prover'. Renaming all existing resources is a lot of work, it will be a slow, gradual transition. During this transition period, please feel free to use both names, for example by writing "formalized with Rocq (formerly Coq)" or "formalized with Coq (also named Rocq)".

Would you prefer the following wording, which adds a mention of the 2024 timeline?

> October 2023: we have decided to rename 'Coq' into 'The Rocq Prover'. We hope to perform the rename in 2024, but renaming all existing resources is a lot of work so it will be a slow, gradual transition.  During this transition period, please feel free to use both names, for example by writing "formalized with Rocq (formerly Coq)" or "formalized with Coq (also named Rocq)".

(Note: I would not recommend any formulation that make it sounds like a complete rename will occur in 2024. This deadline is probably going to be missed, and it's fine, and announcing it too loudly can add pressure to the process.)

File: incl/footer.html
Comment (Zimmi48) [inline incl/footer.html:18]: That's not the official position of the Coq team, which is instead that as long as the renaming hasn't happened, people should still refer to Coq as Coq: https://x.com/CoqLang/status/1746558655758041477

That being said, people are free not to respect this (and in practice, some people have already started referring to Coq as Rocq).

In published papers, my personal recommendation would be to add a footnote on the first mention of Coq to say that there is a planned renaming to "the Rocq prover".

**Review by Zimmi48 — CHANGES_REQUESTED** [review]
I am not sure that the renaming is worth mentioning on the website until it has happened. But if this is deemed worth, then I would put it in the news section like you did (and only there). I would make this piece of news briefer, though.

File: incl/footer.html
Comment (gasche) [reply incl/footer.html:18]: Thanks for the feedback!

>  I would make this piece of news briefer, though.

Could you suggest a wording that you like, here in the comments?

Comment (gasche) [reply incl/footer.html]: @Zimmi48 I made a new iteration that reads as follows (and removes all other changes, as you suggested):

> October 2023: we have decided to rename 'Coq' into 'The Rocq Prover'. The rename is currently in preparation, and will hopefully happen sometime in 2024.

File: incl/footer.html
Comment (Zimmi48) [inline incl/footer.html:15]: Here's my suggestion to provide (hopefully) just the right amount of information:

```suggestion
        <p>The Coq team has decided that Coq will be renamed into 'The Rocq Prover'. Background information available <a href="https://coq.discourse.group/t/coq-community-survey-2022-results-part-iv-and-itp-paper-announcement/2001#renaming-coq-8">here</a>. The rename is currently in preparation, we hope to have a new visual identity and website by the end of 2024, and to do a first release of Rocq around that time.</p>
```

Comment (gasche) [reply incl/footer.html]: Great, thanks! I updated the PR.

**Review by Zimmi48 — APPROVED** [review]
Now the PR looks good to me. I am personally OK to merge this if there is an agreement in @coq/core. This can be obtained by either getting several other core team members approving explicitly this PR and no one rejecting it, or by bringing this to be discussed to the next Coq Call (I won't personally be attending the next Coq Call though, but it could also be the one after).

**Review by proux01 — APPROVED** [review]

**Comment by gasche** [comment]
Thanks everyone!

**Pull request merged**
